pipeline/src/client.py

```diff
@@ -259,6 +263,20 @@ def reprocess(self, start_date=None, end_date=None):
     def create_all_views(self):
         return self.flagged.create_views_all_flags()
 
+    ###########################################################
+
+    def init_flag_dict(self):
+        self.flag_lookup = dict()
+        df = self.flags.get_full_table()
+
+        if df is None:
+            return
+
+        FlagInfo = namedtuple("FlagInfo", "id desc")
```

Comment:
Also the more I think about it the more I don't namedtuple as a construct. My first impression was that it is a clever and concise way to get around Python's lack of structs, but as this snipit illustrates, using them outside of a single file gets cumbersome and unmanageable with scale. How do you feel about adjusting this to be a FlagInfo "struct?"

Reply:
Appreciate the sanity check on that aspect, it seemed reasonable due to the limited use, but your point it well received. I shall make the FlagInfo class inside of flagger.py if that sounds good.

Reply:
That would be greatly appreciated, thank you for changing this.